src/requestHandlers/panel.ts

```diff
@@ -0,0 +1,120 @@
+import { Request, Response } from 'express';
+import * as panelService from '../services/panelService';
+import { genericErrorResponse } from './helpers';
+
+
+interface ResponseObject {
+    success: boolean,
+    body?: any
+    message?: string,
+    error?: string,
+    status?: number,
+}
+
+/**
+ * Creates a panel
+ * @param image //image path
+ * @param index //panel index
+ * @param panel_set_id //id of the panel set the panel is a part of
+ * @returns {ResponseObject}
+ */
+const _createPanelController = async (image: string, index: number, panel_set_id: number) => {
+    try {
+        const response = await panelService.createPanel({
+            image:        image,
+            index:        index,
+            panel_set_id: panel_set_id,
+        });
+
+        return {
+            success: true,
+            body:    response
+        } as ResponseObject;
+    }
+    catch (err) {
+        return genericErrorResponse(err as Error);
+    }
+};
+
+// the actual request 
+const createPanel = async (req: Request, res: Response): Promise<Response> => {
+    const response = await _createPanelController(
+        req.body.image,
+        req.body.index,
+        req.body.panel_set_id,
+    );
```

Comment:
This might be unnecessarily verbose, but if you extracted these body parameters into variables first it could increase readability. Something like
```js
const image: string = req.body.image;
const index: number = Number(req.body.index);
const paneSetId: string = req.body.panel_set_id; // if this is uuid, can't recall

const newPanel = await _createPanelController(image, index, panelSetId);
```

Reply:
With that pattern it would be a bit easier to do validation before the data hits _createPanelController (or formatting, like with the Number() call).